java/arcs/tools/Recipe2Plan.kt

```diff
@@ -0,0 +1,57 @@
+package arcs.tools
+
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.arguments.multiple
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.types.file
+import com.squareup.kotlinpoet.FileSpec
+import java.io.File
+
+/** Generates plans from recipes. */
+class Recipe2Plan : CliktCommand(
+    help = """Generate plans from recipes.
+    
+    This script reads serialized manifests and generates Kotlin files with [Plan] classes.""",
+    printHelpOnEmptyArgs = true
+) {
+    private val outdir by option(help = "output directory; defaults to '.'").file(fileOkay = false)
+    // TODO(b/161994250): Package should be derived from proto
+    private val packageName by option(help = "scope to specified package; default: 'arcs'")
```

Comment:
Might be safer to remove the default. I can't imagine that the `arcs` package is ever the correct package to use, better to make it required and have people supply the correct thing.

Reply:
I'm going to make the default `""`, I want to set it here to prevent the value from being null. You're right, however, that `"arcs"` is not a good choice.

Reply:
Use `option(...).required()` and then it won't be null. 

Running this binary without supplying a package name should be an error, imo.